src/tests/ftest/deployment/target_failure.yaml

```diff
@@ -0,0 +1,72 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+
+timeout: 10M
```

Comment:
FYI, I don't think `launch.py --timeout_multiplier` works with units. I'll let @phender comment

Reply:
Updated to use second.